### PR TITLE
Add Panel Description

### PR DESCRIPTION
**Pull request by MarcSkovMadsen**
## Overview

Adds a description of using Lux and [Panel](https://panel.holoviz.org) together as requested in https://github.com/lux-org/lux/issues/384

## Changes

I've added a Panel section to the `export.rst`  file. 
I've also sorted the data apps sections alphabetically.
My editor has also automatically removed a lot of unnecessary spaces.

## Example Output

image

### CODE REVIEW

File: doc/source/guide/export.rst [hunk 1]
```diff
@@ -293,6 +293,67 @@ If you would like to output HTML be returned directly via :code:`save_as_html`,
...
+  # Can display in notebook
+  lux_panel = to_lux_panel(df)
+
+  # Can display as app with panel serve
+  pn.template.FastListTemplate(title="💡 Lux meets Panel", main=[lux_panel]).servable()
+
+After running :code:`panel serve app.ipynb` or :code:`panel serve app.py` in the command line, you should find the Panel app hosted in your browser (e.g., :code:`http://localhost:5006/app`):
+
+
+.. image:: https://user-images.githubusercontent.com/42288570/122703636-55965980-d252-11eb-9152-38a9df44214d.png
```
Comment (MarcSkovMadsen) [inline doc/source/guide/export.rst:353]: This image should be added to `lux-resources` and the link replaced.

File: doc/source/guide/export.rst [hunk 2]
```diff
@@ -293,6 +293,67 @@ If you would like to output HTML be returned directly via :code:`save_as_html`,
...
+  # Can display in notebook
+  lux_panel = to_lux_panel(df)
+
+  # Can display as app with panel serve
+  pn.template.FastListTemplate(site="💡 Lux and Panel", title="Analysis of Happy Planet Index Dataset", main=[lux_panel]).servable();
+
+After running :code:`panel serve app.ipynb` or :code:`panel serve app.py` in the command line, you should find the Panel app hosted in your browser (e.g., :code:`http://localhost:5006/app`):
+
+
+.. image:: https://user-images.githubusercontent.com/42288570/122705585-56c98580-d256-11eb-9bd6-a259cc2af5d6.png
```
Comment (MarcSkovMadsen) [inline doc/source/guide/export.rst:353]: This image should be added to `lux-resources` and the url updated.

Comment (micahtyong) [reply doc/source/guide/export.rst:353]: I've just added the image to lux-resources. Please replace line 353 with 
```
.. image:: https://github.com/lux-org/lux-resources/blob/master/doc_img/export-panel.png?raw=true
```

Comment (MarcSkovMadsen) [reply doc/source/guide/export.rst]: Done @micahtyong

**Comment by MarcSkovMadsen** [comment]
I've now replaced the image url as requested.

**Comment by dorisjlee** [comment]
Thanks for your contribution @MarcSkovMadsen! I just tested this out and the Panel integration is really cool!
I made a minor change to save_as_html so that the Lux logo links to a new tab, otherwise, your changes look great!

**Review by dorisjlee — APPROVED** [review]

**Comment by MarcSkovMadsen** [comment]
Thanks @dorisjlee 👍

**Pull request merged**